app/src/test/java/com/ikarabulut/BoardTest.java

```diff
@@ -69,4 +69,121 @@ void setRandomGrid() {
         assertFalse(isDeadBoard);
     }
 
-}
\ No newline at end of file
+    @Test
+    @DisplayName("A cell that is dead can be set to alive")
+    void setAliveAt() {
+        Board board = new Board();
+
+        Boolean cellStatus = board.isCellAliveAt(1, 1);
+        assertFalse(cellStatus);
+
+        board.setAliveAt(1, 1);
+        cellStatus = board.isCellAliveAt(1, 1);
+
+        assertTrue(cellStatus);
+
+    }
+
+    @Nested
+    @DisplayName("Tests for getAliveNeighborsCountOf")
```

Comment:
```suggestion
    @DisplayName("getAliveNeighborsCountOf()")
```

Reply:
I kinda like leaving 'Tests for' cause it makes it more easily readable in the test output 
![Screen Shot 2022-05-05 at 4 45 51 PM](https://user-images.githubusercontent.com/61036790/167030858-15672b3f-4518-4f96-bdf2-bc54c35e10d8.png)

Reply:
Actually its obvious that these are tests so there is no need to call that out. Ill update it.